docs/finders/advanced.mdx

```diff
@@ -81,6 +81,50 @@ You can also change the timeout ad-hoc:
 await $(#addComment).tap(findTimeout: Duration(seconds: 30));
 ```
 
+### You gotta pump it up! But which one to use?
+
+Pumpimg is a mechanism provided by Flutter's engine that help us determine, 
+if we can execute the next action during the test. It is an equivalent 
```

Comment:
This is a great paragraph.

I have some minor ideas for improvements but it'll be faster if I just make them, instead of writing all of them here.